Gopkg.toml

```diff
@@ -35,33 +35,33 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.11.0"
 
 [[override]]
   name = "github.com/golang/protobuf"
   version = "v1.1.0"
 
 [[constraint]]
   name = "github.com/knative/eventing"
-  revision = "f3ce0d1847b51a47918870ec1dea7871e813a8c6" # Awaiting proper tag / blessed sha
+  revision = "f8e956305ed4bc07220d48905b0f1045a0043693" # Awaiting proper tag / blessed sha
 
 [[constraint]]
   name = "github.com/knative/serving"
-  version = "v0.1.0"
+  revision = "3da9b7a56f155567b15ac46aad16c96a17972e50"
 
 [[constraint]]
   name = "github.com/knative/build"
   # This is v0.1.0, but dep doesn't know how to detect that. Can't use v0.1.0 because serving has a dependency against
```

Comment:
Update comment for knative/eventing,serving,build as to where the ref spec is coming from